Lab 3/README.md

```diff
@@ -10,7 +10,7 @@ In this lab, set up an application to leverage the Watson Tone Analyzer service.
 2. Change the directory to Lab 3/watson.
 
 3. Build the `watson` image:
-   ```docker build -t registry.ng.bluemix.net/<namespace>/watson```
+   ```docker build -t registry.ng.bluemix.net/<namespace>/watson .```
```

Comment:
These two in lab 3 were my main concerns. Is there any particular reason to be `cd` into each directory?

If not you can skip those and do 
`docker build -t registry.ng.bluemix.net/<namespace>/watson ./watson`
`docker build -t registry.ng.bluemix.net/<namespace>/watson-talk ./watson-talk`